packages/oasis-actions/src/strategies/aave/adjust.ts

```diff
@@ -36,10 +36,22 @@ export async function adjust(
   const depositDebtAmountInWei = args.depositedByUser?.debtInWei || ZERO
   const depositCollateralAmountInWei = args.depositedByUser?.collateralInWei || ZERO
 
-  const estimatedSwapAmount = amountToWei(new BigNumber(1))
-
   const currentPosition = dependencies.currentPosition
 
+  let isIncreasingRisk = true
+  if (
+    new RiskRatio(multiple, RiskRatio.TYPE.MULITPLE).loanToValue.lte(
+      currentPosition.riskRatio.loanToValue,
+    )
+  ) {
+    isIncreasingRisk = false
+  }
+
+  const estimatedSwapAmount = amountToWei(
+    new BigNumber(1),
+    isIncreasingRisk ? args.debtToken.precision : args.collateralToken.precision,
+  )
+
```

Comment:
Maybe instead of checking `isIncreasingRisk` there should be just one please where you assign `fromToken` and `toToken`

Reply:
Have removed most of the flags and just working with from and to